docs/src/tutorial.md

```diff
@@ -35,18 +35,52 @@ norm(A \ v - op * v) / norm(v)
 ```
 In this example, the Cholesky factor is computed only once and can be used many times transparently.
 
+## mul!
+
+It is often useful to reuse the memory used by the operator. 
+For that use, we can use `mul!` as if we were using matrices.
+We may wait to reuse the vectors preallocated by the operator in order to save memory
+
+```@example ex2
+using LinearOperators, LinearAlgebra # hide
+m, n = 50, 30
+A = rand(m, n) + im * rand(m, n)
+op = LinearOperator(A)
+v = rand(n)
+al = @allocated mul!(op.Mv, op, v) # op * v, store result in op.Mv
```

Comment:
Is this the intended use? It seems a bit odd to ask the user to access a temporary array stored inside `op`. Maybe the example should preallocate a new vector for storage?

Reply:
The user could also use a new vector, but you're right I will add more documentation about this